File: dx-api/app/consts/feedback_type.go
package consts

// Feedback type values.
const (
	FeedbackTypeFeature = "feature"
	FeedbackTypeContent = "content"
	FeedbackTypeUX      = "ux"
	FeedbackTypeBug     = "bug"
	FeedbackTypeOther   = "other"
)

// FeedbackTypeLabels maps each feedback type to its Chinese label.
var FeedbackTypeLabels = map[string]string{
	FeedbackTypeFeature: "功能建议",
	FeedbackTypeContent: "内容纠错",
	FeedbackTypeUX:      "界面体验",
	FeedbackTypeBug:     "Bug 报告",
	FeedbackTypeOther:   "其它",
}

File: dx-web/src/consts/feedback-type.ts
export const FEEDBACK_TYPES = {
  FEATURE: "feature",
  CONTENT: "content",
  UX: "ux",
  BUG: "bug",
  OTHER: "other",
} as const;

export type FeedbackType = (typeof FEEDBACK_TYPES)[keyof typeof FEEDBACK_TYPES];

export const FEEDBACK_TYPE_LABELS: Record<FeedbackType, string> = {
  feature: "功能建议",
  content: "内容纠错",
  ux: "界面体验",
  bug: "Bug 报告",
  other: "其它",
};
